packages/voter-dapp/src/ResolvedRequests.js

```diff
@@ -84,8 +84,9 @@ function ResolvedRequests({ votingAccount }) {
   const newProposalEvents = useCacheEvents(
     "Governor",
     "NewProposal",
-    useMemo(() => ({ fromBlock: 0 }))
+    useMemo(() => ({ fromBlock: 0 }), [])
```

Comment:
This is what was causing infinite loop

Reply:
Why does this fix the infinite loop? Really elegant solution if so

Reply:
i think because if you dont unclude the `[]` it will run every render, and its probably causing a rerender when it runs. by adding the `[]` it means just run once.

Reply:
I think the second arg to any of the `useX` hooks is an array of dependencies, so I guess if you leave out the array of dependency then it will always re-render. If you explicitly set the dependencies to non with `[]` then it should only render once.

Reply:
yes thats my understanding and explained a lot better